src/main/resources/application.yml

```diff
@@ -48,13 +48,20 @@ oauth:
 
 identity:
   identityAPIUrl: "${oauth.serviceUrl}/api/identities"
+  identityWhiteListUrl: "${oauth.serviceUrl}/domain/isWhitelisted"
 
 govNotify:
   enabled: ${GOV_NOTIFY_ENABLED:false}
   key: ${GOV_NOTIFY_API_KEY:changeme}
   template:
     lineManager: ${GOV_NOTIFY_LINEMANAGER_TEMPLATE_ID:changeme}
 
+#Validation properties
+agencyToken:
+  capacity:
+    max: 1500
```

Comment:
These will need to be configurable at some point

Reply:
The application.yml does mean they are configurable, although it depends on the DevOps.  I am not familiar with the DevOps on this project.  Therefore please can you explain what you mean by configurable.

Reply:
Agree its a bit pointless, but to be configurable via env variable has been requested in requirements: 
e.g. ${AGENCY_TOKEN_MAX_CAPACITY:1500}

Reply:
ah ok, i see what you mean.  Thank you.  Ill add it.

Reply:
Updated